tests/util/addons.sh

```diff
@@ -15,6 +15,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# dummy comment to force test ALL
```

Comment:
Remove before merge as this is temporary to cause all tests to be run.

Reply:
This shouldn't be needed since go.mod is one of the updated files which will also force test ALL.
https://github.com/istio/istio.io/blob/9f2b9850ac899ce50a93ac94a626046ed320aabf/scripts/pr_tests.py#L94